src/bp/core/logger/index.ts

```diff
@@ -1,3 +1,7 @@
-export * from './logger'
-export * from './db-persister'
-export * from './file-persister'
+export * from './persisted-console-logger'
+export * from './logger-db-persister'
+export * from './logger-file-persister'
```

Comment:
I find the name of those files really confusing. Maybe we could move them into a subfolder like `persisted` and then rename name accordingly. Or we could simply rename this file: `persisted-console-logger` into `logger`. I think just this change would be enough. 

What do you think?